priv/template/test/support/repo_case.ex.eex

```diff
@@ -0,0 +1,25 @@
+<%= if @ecto do %>defmodule <%= @module %>.RepoCase do
+  use ExUnit.CaseTemplate
+  # SEE https://hexdocs.pm/ecto/testing-with-ecto.html for more information
+
+  using do
+    quote do
+      # alias <%= @module %>.Repo
```

Comment:
Is this just importing commented out things?

Reply:
Yeah, I don't want to impart the aliases and imports on people, but that's the configuration recommended by Ecto docs, so I kept it here as a jump off point and a token of good will ;)

Reply:
fair enough